py/server/deephaven/table_factory.py

```diff
@@ -342,12 +342,12 @@ def ring_table(parent: Table, capacity: int, initialize: bool = True) -> Table:
         raise DHError(e, "failed to create a ring table.") from e
 
 
-def function_generated_table(table_generator: Callable[[], Table],
+def function_generated_table(table_generator: Callable[..., Table],
            source_tables: Union[Table, List[Table]] = None,
            refresh_interval_ms: int = None,
-           exec_ctx: ExecutionContext = None) -> Table:
-    """
-    Creates an abstract table that is generated by running the table_generator() function. The function will first be
+           exec_ctx: ExecutionContext = None,
+           **kwargs) -> Table:
```

Comment:
@jmao-denver should we support `*args` and `**kwargs` or is the latter the standard for something like this?
https://stackabuse.com/variable-length-arguments-in-python-with-args-and-kwargs/

Reply:
I think requiring keyword arguments only isn’t too much of a burden for the users. Moving the parameters around in the factory function function_generated_table to support positional args for the table_generator function might hurt the usability of function_generated_table